linked.js

```diff
@@ -0,0 +1,155 @@
+class Node {
+    constructor(value = null, next = null) {
+        this.value = value;
+        this.next = next
+    }
```

Comment:
Consider
```suggestion
    constructor(value) {
        this.value = value;
        this.next = null
    }
```

None of the uses of Node below provides `next` at initialisation.